app/views/admin/stats/top_balances/_channels.html.slim

```diff
@@ -0,0 +1,20 @@
+- table_identifier = "youtube#channel"
```

Comment:
IMO These things should not be in the view

Reply:
It's localized logic. There's no existing constant elsewhere unless I do `YoutubeChannelDetails::YOUTUBE_PREFIX.chomp`. Thoughts?

Reply:
No, sorry I mean querying the database for these things. It violates the separation of MVC and it can make finding queries a little harder